session/src/main/java/org/apache/iotdb/session/Session.java

```diff
@@ -558,6 +558,15 @@ public TSStatus createTimeseries(String path, TSDataType dataType,
     }
   }
 
+  public boolean checkTimeseriesExists(String path) throws IoTDBSessionException {
```

Comment:
This is not necessary in this PR, right? [1]
https://github.com/apache/incubator-iotdb/pull/824/files

Reply:
Updated.